tests/integration_tests.py

```diff
@@ -0,0 +1,49 @@
+import requests
+
+
+class PetStore:
+    """Petstore class.
+
+    Attributes:
+        url: Petstore API base URL.
+    """
+    def __init__(self):
+        """Class constructor method."""
+        self.url = "http://localhost:80/pets"
+
+    def add_path_to_url(self, id):
+        """Return petstore base URL with appended path"""
+        return "/".join([self.url, str(id)])
```

Comment:
Replaced the `+` string concatenation operator with the `.join()` method because it is more performant. It's of course utterly negligible here, but guess it's just best to entirely forget about being able to use `+` to concat strings altogether unless there is a HUGE readability improvement that does more than offset the loss in performance.